app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -97,6 +97,7 @@ def save_in_person_enrollment(user, profile)
           profile: profile,
           user: user,
           current_address_matches_id: user_session.dig(:idv, :applicant, :same_address_as_id),
+          unique_id: InPersonEnrollment.generate_unique_id,
```

Comment:
This will fail if we do hit a duplicate ID value, would be nice to retry in that case. Could come in a different PR or ticket though

Reply:
In fact, I can add this as part of the PR that I'm working on now if you'd like

Reply:
Follow up ticket created: https://cm-jira.usa.gov/browse/LG-7024